Wrappers/Python/cil/optimisation/utilities/__init__.py

```diff
@@ -19,3 +19,5 @@
 
 from .sampler import Sampler
 from .sampler import SamplerRandom
+from .StepSizeMethods import ConstantStepSize, ArmijoStepSize
+from .preconditioner import  Preconditioner, AdaptiveSensitivity, Sensitivity, Adam, AdaGrad
```

Comment:
Do we want to have `Adam` `AdaGrad` as preconditioners or as algorithms?

Reply:
Could question! I think it is probably sensible to remove them from this PR but I had fun writing them!

Reply:
I think this ties back to the question what name we use. If we use `Preconditioner`, it makes little sense to derive Adam from it.